### DISCUSSION TITLE
Working with no display?

### DISCUSSION THREAD

**Discussion by dp-alvarez** [DavidS95/Smokeless_UMAF#24] [General]
I'm trying it out on a Thinkpad T14s gen1 (Zen2 4750u processor) and there is some strange behavior.
When booting from this I get some text diagnostic and then a black screen, but it seems like it is actually working since I can access the boot menu or boot directly to the OS by blindly trying to navigate the menu.
Of course it is not usable for actually setting any UEFI settings with no display.

I tried using an external display and messing around on the UEFI settings but unsurprisingly it didn't work.
Has anyone gotten it to work on a similar platform, or any tips on where one would start troubleshooting this?

**Comment by PetterPet01** [comment]
Have you been able to resolve it? I have the same laptop with the exact same issues.

**Reply by dp-alvarez** [reply]
Nothing so far. If you are looking for CPPC support then there is a thread for that in the Lenovo forums at https://forums.lenovo.com/t5/Other-Linux-Discussions/amd-pstate-driver-support-for-AMD-laptops/m-p/5135917?page=8

Someone said that even if one could get this tool to work, there are missing bits in the bios that would prevent it from working.
Unfortunately I think we are going to be stuck with cpufreq on this platform.

**Reply by ff66theone** [reply]
I have a Lenovo V14 G4 here, and it works fine from USB but gives a blank screen when installed on the HDD (booted from a custom GRUB entry on the ESP)
I'll try to disable every "legacy support" settings, since that's all I changed between the USB method and HDD method (for a side project that is making DOS 3.31 work on that same laptop).

[EDIT]
After changing all these options in the BIOS, I can tell you that my HDD installation is working again as intended. You should try to disable compatibility features on your side as well.
